src/lib/Scenes/Collection/Screens/CollectionArtworks.tsx

```diff
@@ -0,0 +1,67 @@
+import { CollectionArtworks_collection } from "__generated__/CollectionArtworks_collection.graphql"
+import { InfiniteScrollArtworksGridContainer as InfiniteScrollArtworksGrid } from "lib/Components/ArtworkGrids/InfiniteScrollArtworksGrid"
+import { get } from "lib/utils/get"
+import React from "react"
+import { createPaginationContainer, graphql, RelayPaginationProp } from "react-relay"
+
+export const CollectionArtworks: React.FC<{
```

Comment:
This component is essentially just a wrapper for the InfiniteScrollArtworksGrid?